packages/main/src/components/DynamicPage/DynamicPage.stories.tsx

```diff
@@ -0,0 +1,220 @@
+import '@ui5/webcomponents-icons/dist/icons/action';
```

Comment:
I would prefer having the demo in mdx format. MDX will be our documentation format going forward